pages/common/heroku.md

```diff
@@ -0,0 +1,27 @@
+# heroku
+
+> Create and manage Heroku apps from the command line.
+
+- Login to your heroku account:
+
+`heroku login`
+
+- Create a heroku app:
+
+`heroku create`
+
+- Show logs for an app:
+
+`heroku logs --app {{app_name}}`
+
+- Run a one-off process inside a heroku virtual machine:
+
+`heroku run {{process_name}} --app {{app_name}}`
+
+- List dynos for an app:
```

Comment:
Dyno actually means a lightweight Linux container that runs a single user-specified command. Its not a proper virtual machine as it can't run multiple processes. What word should I use in place of dyno? Will "virtual machine" be allright? @agnivade